src/builder/build_traits.rs

```diff
@@ -190,7 +201,7 @@ pub trait Dataflow: Container {
         op: impl Into<OpType>,
         input_wires: impl IntoIterator<Item = Wire>,
     ) -> Result<BuildHandle<DataflowOpID>, BuildError> {
-        self.add_dataflow_node(NodeType::pure(op), input_wires)
+        self.add_dataflow_node(NodeType::open_extensions(op), input_wires)
```

Comment:
I'm gonna stop commenting about things like this that look a bit out of date until you've done some merging/rebasing/something....